pinterestClone/ImagesListFolder/SingleImageFolder/SingleImageViewController.swift

```diff
@@ -88,6 +93,24 @@ final class SingleImageViewController: UIViewController {
         scrollView.setContentOffset(CGPoint(x: x, y: y), animated: false)
     }
     
+    func fetchFullImage() {
+        guard let fullImageUrl = fullImageUrl else {return}
+        UIBlockingProgressHUD.show()
+        imageView.kf.setImage(with: fullImageUrl) {[weak self] result in
+            guard let self = self else {return}
+            UIBlockingProgressHUD.dismiss()
+            switch result {
+            case .success(let ResultImage):
+                createSingleImageView()
+                self.rescaleAndCenterImageInScrollView(image: ResultImage.image)
```

Comment:
последи за camelCase в своем коде:
```suggestion
            case .success(let resultImage):
                createSingleImageView()
                self.rescaleAndCenterImageInScrollView(image: resultImage.image)
```